jans-fido2/server/src/main/java/io/jans/fido2/service/mds/MdsService.java

```diff
@@ -1,21 +1,3 @@
-/*
```

Comment:
We need to put:
```
* Janssen Project software is available under the MIT License (2008). See http://opensource.org/licenses/MIT for full text.
 *
 * Copyright (c) 2020, Janssen Project
 */
 ```

Reply:
There is same question about log messages which @jgomer2001 raised in previous file